src/Components/SelectLanguagePage/SelectLanguagePage.tsx

```diff
@@ -0,0 +1,77 @@
+import { FormControl, Select, InputLabel, MenuItem, SelectChangeEvent, Button, Box } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+import { Context } from '../Wrapper/Wrapper.tsx';
+import { useContext, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import languageOptions from '../../Assets/languageOptions.tsx';
+
+const SelectLanguagePage = () => {
+  const context = useContext(Context);
+  const { formData } = context;
+  const queryString = formData.immutableReferrer ? `?referrer=${formData.immutableReferrer}` : '';
+  const navigate = useNavigate();
+
+  const createMenuItems = (optionList: Record<string, string>, disabledFMId: string, disabledFMDefault: string) => {
+    const disabledSelectMenuItem = (
+      <MenuItem value="disabled-select" key="disabled-select" disabled>
+        <FormattedMessage id={disabledFMId} defaultMessage={disabledFMDefault} />
+      </MenuItem>
+    );
+    const menuItemKeys = Object.keys(optionList);
+    const menuItemLabels = Object.values(optionList);
+
+    const dropdownMenuItems = menuItemKeys.map((option, i) => {
+      return (
+        <MenuItem value={option} key={option}>
+          {menuItemLabels[i]}
+        </MenuItem>
+      );
+    });
+
+    return [disabledSelectMenuItem, dropdownMenuItems];
+  };
+
+  useEffect(() => {
+    const continueOnEnter = (event: Event) => {
+      if (event.key === 'Enter') {
+        navigate(`/step-2${queryString}`);
+      }
+    };
+    document.addEventListener('keyup', continueOnEnter);
+    return () => {
+      document.removeEventListener('keyup', continueOnEnter); // remove event listener on onmount
+    };
+  });
+
+  return (
+    <main className="benefits-form">
+      <h1 className="sub-header">
+        <FormattedMessage
+          id="selectLanguage.header"
+          defaultMessage="Before you begin, what is your preferred language?"
+        />
+      </h1>
+      <FormControl sx={{ width: '150px' }}>
+        <InputLabel id="language-select-label">
+          <FormattedMessage id="selectLang.text" defaultMessage="Language" />
```

Comment:
We will need to add this new FormattedMessage's `id` & `defaultMessage` to the backend via the translations UI.